packages/libs/eda/src/lib/core/components/filter/HistogramFilter.tsx

```diff
@@ -654,122 +656,148 @@ function HistogramPlotWithControls({
       />
 
       <div style={{ display: 'flex', flexDirection: 'row' }}>
-        <LabelledGroup label="X-axis controls">
-          <BinWidthControl
-            binWidth={data?.binWidthSlider?.binWidth}
-            binWidthStep={data?.binWidthSlider?.binWidthStep}
-            binWidthRange={data?.binWidthSlider?.binWidthRange}
-            binUnit={uiState.binWidthTimeUnit ?? 'year'}
-            binUnitOptions={
-              data?.binWidthSlider?.valueType === 'date'
-                ? ['day', 'week', 'month', 'year']
-                : undefined
-            }
-            onBinWidthChange={handleBinWidthChange}
-            valueType={data?.binWidthSlider?.valueType}
-            containerStyles={{ minHeight: widgetHeight }}
-          />
-
-          <AxisRangeControl
-            label="Range"
-            range={uiState.independentAxisRange}
-            onRangeChange={handleIndependentAxisRangeChange}
-            valueType={data?.binWidthSlider?.valueType}
-            containerStyles={{ minWidth: '400px' }}
-          />
-          {/* truncation notification */}
-          {truncatedIndependentAxisWarning ? (
-            <Notification
-              title={''}
-              text={truncatedIndependentAxisWarning}
-              // this was defined as LIGHT_BLUE
-              color={'#5586BE'}
-              onAcknowledgement={() => {
-                setTruncatedIndependentAxisWarning('');
-              }}
-              showWarningIcon={true}
-              containerStyles={{
-                maxWidth:
-                  data?.binWidthSlider?.valueType === 'date'
-                    ? '34.5em'
-                    : '38.5em',
-              }}
-            />
-          ) : null}
-          <Button
-            type={'outlined'}
-            text={'Reset X-axis to defaults'}
-            onClick={handleIndependentAxisSettingsReset}
-            containerStyles={{
-              paddingTop: '1.0em',
-              width: '50%',
-              float: 'right',
+        <div style={{ display: 'flex', flexDirection: 'column' }}>
+          {/* set Undo icon and its behavior */}
+          <div
+            style={{
+              display: 'flex',
+              flexDirection: 'row',
+              alignItems: 'center',
             }}
-          />
-        </LabelledGroup>
+          >
+            <LabelledGroup label="X-axis controls"> </LabelledGroup>
```

Comment:
Does this still need to be a LabelledGroup? I think this can just be a heading?

Reply:
@asizemore We are using the LabelledGroup for all Visualizations as well. Technically it is not necessary to use it but for consistency it may need to be changed overall. Akin to above, this may be handled at another ticket but it is more like a technical debt level, I suppose.